game_stats_fixtures/games_fixtures.csv

```diff
@@ -0,0 +1,20 @@
+game_id,season,type,date_time,away_team_id,home_team_id,away_goals,home_goals,venue,venue_link
```

Comment:
@ajtran303 I went through and added all of your games_fixtures from your branch so it should match your file exactly